core/modules/views/includes/cache.inc

```diff
@@ -127,8 +127,25 @@ function _views_data_process_entity_types(&$data) {
 function _views_fetch_plugin_data($type = NULL, $plugin = NULL, $reset = FALSE) {
   static $cache = NULL;
   if (!isset($cache) || $reset) {
-    views_include_handlers();
-    $cache = views_discover_plugins();
+    // Load necessary code once.
+    if (!isset($cache)) {
+      views_include('plugins');
+      views_include_handlers();
+    }
+    // Because plugin data contains translated strings, and as such can be
+    // expensive to build, the results are cached per language.
+    global $language;
+    $cache_key = 'views:plugin_data:' . $language->langcode;
```

Comment:
Wondering if we should be checking if `$language->langcode` is set here 🤔

Reply:
It is always set  on a full bootstrap. I think this is safe.